src/mono/wasm/runtime/diagnostics.ts

```diff
@@ -82,30 +82,162 @@ class EventPipeFileSession implements EventPipeSession {
     }
 }
 
+const eventLevel = {
```

Comment:
Why not const enum?

Reply:
Const enum gets completely erased by the typescript compiler - but I actually want `MONO.diagnostics.EventLevel` in JS to have a property for each of the values here.  (So something like https://www.typescriptlang.org/docs/handbook/enums.html#objects-vs-enums)